config/scylladb-sink-quickstart.properties

```diff
@@ -4,6 +4,8 @@ topics=<comma-separated-kafka-topics-name>
 tasks.max=1
 connector.class=io.connect.scylladb.ScyllaDbSinkConnector
 scylladb.contact.points=<scylladb-hosts>
+#scylladb.contact.points=<JSON-String-with-private network address:port:external-network-address:port>
+#Eg. {\"10.0.24.69:9042\": \"sl-eu-lon-2-portal.3.dblayer.com:15227\", \"10.0.24.71:9042\": \"sl-eu-lon-2-portal.2.dblayer.com:15229\", \"10.0.24.70:9042\": \"sl-eu-lon-2-portal.1.dblayer.com:15228\"}
```

Comment:
I find this example very confusing - we should always mention the straight forward use-cases first and then the more complicated cases later.